packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -42,27 +56,178 @@
     get_name,
 )
 from packages.valory.skills.abstract_round_abci.utils import (
-    DEFAULT_TENDERMINT_P2P_URL,
     get_data_from_nested_dict,
     get_value_with_type,
 )
 
 
 NUMBER_OF_RETRIES: int = 5
-DEFAULT_BACKOFF_FACTOR: int = 2
+DEFAULT_BACKOFF_FACTOR: float = 2.0
 DEFAULT_TYPE_NAME: str = "str"
-_DEFAULT_REQUEST_TIMEOUT = 10.0
-_DEFAULT_REQUEST_RETRY_DELAY = 1.0
-_DEFAULT_TX_TIMEOUT = 10.0
-_DEFAULT_TX_MAX_ATTEMPTS = 10
-_DEFAULT_CLEANUP_HISTORY_DEPTH_CURRENT = None
```

Comment:
This is better defined on the skill.yaml. Then they can actually be overridden at the agent level.